### ISSUE TITLE
🐛 BUG: 'Welcome to squad' message isn't populated from mobile notification

### ISSUE THREAD

**Issue by frank-connolly** [dailydotdev/daily#1245]
### What went wrong? 🤔

As a squad admin, when I get a 'someone joined your squad welcome them' notification. On desktop clicking it brings me to comment on welcome post autopopulated with their tag and welcome. On mobile it's blank.

### Expected Behavior

Mobile behaviour should match desktop.

### Steps to Reproduce Issue

```bash
1. Go to 'X joined your squad' notification
2. Click on the notification on desktop and repeat on mobile
3. See the difference in the comment placeholder
```

### Solution Proposed

Review desktop event on notification link. Mirror it on mobile.

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Browsers

Chrome

### OS

Windows

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @frank-connolly,

Thanks for the report! We'll look into it.

Sidenote: do you find this call to action useful as a squad admin?

**Comment by frank-connolly** [comment]
> Hi @frank-connolly,
> 
> Thanks for the report! We'll look into it.
> 
> Sidenote: do you find this call to action useful as a squad admin?

Yes people seem to appreciate a welcome note and I'm not always at my desktop.

**Comment by sshanzel** [comment]
Thank you for reporting! 🥳 

The issue is most likely within the `CommentModal` which is used on mobile only. The `initialContent` from the props is probably not being utilized at the moment. Incorporating it within the component itself should most likely fix the issue. For anyone interested 👀

**Comment by rebelchris** [comment]
This is now fixed
